src/cfp/community-booth.njk

```diff
@@ -0,0 +1,118 @@
+---
+title: Community Booth Program
+description: |
+    PyCon India 2025 invites community groups to apply for a community booth, an excellent opportunity to engage with the dynamic  Python enthusiasts   across India. A community booth helps to showcase your work, connect with like-minded folks, and grow your network. It’s a great way to share your story, promote your initiatives, and inspire more contributors to join your cause.
```

Comment:
Since we moved away from Markdown and doing it in njk, we will need to copy over the header from other njk files. This description thing won't work sadly.